tmt/steps/provision/__init__.py

```diff
@@ -912,6 +912,10 @@ def setup(self) -> None:
 
         Setup the guest after it has been started. It is called after :py:meth:`Guest.start`.
         """
+        from tmt.steps.execute.internal import effective_pidfile_root
+        pid_directory = effective_pidfile_root()
+        self.execute(ShellScript(f"mkdir -p {pid_directory}"))
+        self.execute(ShellScript(f"chmod ugo+rwx {pid_directory}"))
```

Comment:
Ha, one more thing I just realized: we can run both commands in one step, safe one needless SSH connection: `self.execute(ShellScript(f"mkdir -p {pid_directory} && chmod ugo+rwx {pid_directory}"))`

Reply:
Updated, thanks !